app/src/main/java/com/github/sdpteam15/polyevents/view/activity/activityprovider/EventManagementActivityProvider.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.sdpteam15.polyevents.view.activity.activityprovider
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.github.sdpteam15.polyevents.R
+
+class EventManagementActivityProvider : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_event_management_provider)
+    }
+}
```

Comment:
Do we still need this class ?

Reply:
Yes, it will be used to display the status of an event edit request for the event organiser (like for the material requests)